subgraphs/inventory/Dockerfile

```
@@ -14,4 +15,13 @@ COPY app .
 # Do the actual build
 RUN gradle clean build --no-daemon --parallel
 
-CMD ["java", "-jar", "build/libs/app.jar"]
+
+# Create a new layer with the build output to run the app
+FROM openjdk:11-jre-slim
+ENV ARTIFACT_NAME=app.jar
+
+WORKDIR /usr/src/app
+
+COPY --from=TEMP_BUILD_IMAGE /usr/src/app/build/libs/$ARTIFACT_NAME .
+
+ENTRYPOINT ["java", "-jar", "-noverify", $ARTIFACT_NAME]
```

This is [failing at runtime](https://github.com/apollographql/supergraph-demo-fed2/pull/78/checks#step:8:22) likely b/c we're using exec form for [fast graceful shutdown](https://joostvdg.github.io/docker/graceful-shutdown/) (not shell form) but something like this will work:

```suggestion
ENTRYPOINT exec "java" "-jar" "-noverify" "${ARTIFACT_NAME}"
```